force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -66,6 +66,69 @@
         <shortDescription>Anonymous</shortDescription>
         <value>Anonymous</value>
     </labels>
+    <labels>
+        <fullName>BSDT_Group_Option_Title</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for the Bulk Service Delivery Tool Group Data Entry Option</shortDescription>
+        <value>Create Service Deliveries for a Group of Contacts</value>
+    </labels>
+    <labels>
+        <fullName>BSDT_Group_Option_Description</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for the Bulk Service Delivery Tool Group Data Entry Option</shortDescription>
+        <value>Enter service deliveries for a group of contacts. For example a workshop or a class.</value>
```

Comment:
capital S and D on Service Deliveries cuz it's an object? (tagged jess on the same q below.) Also, could we make the example here explicitly a drop-in scenario? "For example, a drop-in session." ?